model_compression_toolkit/qat/pytorch/quantizer/lsq/uniform_lsq.py

```diff
@@ -12,66 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-from typing import Union
 import numpy as np
 import torch
 import torch.nn as nn
 
-from model_compression_toolkit.constants import RANGE_MAX, RANGE_MIN
-from model_compression_toolkit.trainable_infrastructure.common.constants import FQ_MIN, FQ_MAX
-
-from model_compression_toolkit.qat import TrainingMethod
-from model_compression_toolkit.target_platform_capabilities.target_platform import QuantizationMethod
 from mct_quantizers import QuantizationTarget, PytorchQuantizationWrapper
-from model_compression_toolkit import constants as C
-
-from model_compression_toolkit.qat.pytorch.quantizer.base_pytorch_qat_quantizer import BasePytorchQATTrainableQuantizer
 from mct_quantizers import mark_quantizer
-from model_compression_toolkit.qat.pytorch.quantizer.quantizer_utils import ste_round, grad_scale
-from model_compression_toolkit.core.pytorch.utils import to_torch_tensor
 from mct_quantizers.pytorch.quantizers import \
-    WeightsUniformInferableQuantizer, ActivationUniformInferableQuantizer
-from model_compression_toolkit.trainable_infrastructure.common.trainable_quantizer_config import \
-    TrainableQuantizerWeightsConfig, TrainableQuantizerActivationConfig
+    WeightsUniformInferableQuantizer
+
+from model_compression_toolkit.constants import RANGE_MAX, RANGE_MIN
```

Comment:
how do you sort the imports like that? I'm sure it's not manually right?

Reply:
PyCharm-> Code -> Optimize Imports